apps/yara/find_extender.h

```diff
@@ -389,48 +285,121 @@ extend(Extender<THaystack, TNeedle, TDistance, TSpec> & extender,
     THaystackSize haystackLength = length(extender.haystack[haystackId]);
     TNeedlePos needleLength = length(needle);
 
+    TErrors matchErrors = 0;
+
+#ifdef YARA_PRINT_ALIGN
+    std::cerr << std::endl << "======================================================================" << std::endl;
+    std::cerr << "SEED: " << Pair<THaystackPos>(haystackBegin, haystackEnd) << " -- " << Pair<TNeedlePos>(needleBegin, needleEnd) << std::endl;
+
+    {
+        Align<Dna5String, AnchorGaps<>> align;
+        resize(rows(align), 2);
+        assignSource(row(align, 0), infix(extender.haystack, haystackBegin, haystackEnd));
```

Comment:
Probably irrelevant as this seems like a debug function, but in fact here the infix will be copied in the gaps structure thanks to our glory `Holder` class.
Just want to make sure, that this might cause a performance bottleneck.

Reply:
you are right, it's for debugging only.